clients/bazel-module-registry/src/main/kotlin/RemoteBazelModuleRegistryService.kt

```diff
@@ -26,19 +26,20 @@ import org.apache.logging.log4j.kotlin.logger
 
 import retrofit2.Retrofit
 import retrofit2.converter.kotlinx.serialization.asConverterFactory
-import retrofit2.http.GET
-import retrofit2.http.Path
 
 /**
  * The client uses the Bazel Central Registry by default.
  */
 const val DEFAULT_URL = "https://bcr.bazel.build"
 
 /**
- * Interface for a Bazel Module Registry, based on the directory structure of https://bcr.bazel.build/ and the Git
- * repository it is based on (https://github.com/bazelbuild/bazel-central-registry/).
+ * A remote Bazel Module Registry, based on the directory structure of https://bcr.bazel.build/ and the Git repository
+ * it is based on (https://github.com/bazelbuild/bazel-central-registry/).
```

Comment:
Also here, while rewording the docs anyway, please incorporate the `client` and `baseUrl` parameters to explain their meaning.